### PR TITLE
Menu: set_property_widget() should take IsA<Widget> instead of Widget

### LINKED ISSUE #846: MenuExt::set_property_attach_widget() should take IsA<Widget> instead of Widget

### PR DESCRIPTION
**Pull request by Cogitri**
fixes #846

### CODE REVIEW

File: Gir.toml [hunk 1]
```diff
@@ -1334,6 +1334,9 @@ manual_traits = ["GtkMenuExtManual"]
     [[object.function]]
     name = "place_on_monitor"
     version = "3.22"
+    [[object.function]]
+    name = "set_property_attach_widget"
+    ignore = true
```
Comment (Cogitri) [inline Gir.toml]: I can't tell `gir` to take a different param, can I? After reading the readme there didn't seem to be a way to do it, but maybe I'm just missing something?

Comment (Cogitri) [reply Gir.toml]: Also, did I typo here? gir still generates the function

**Comment by sdroege** [comment]
I believe this is a bug in gir. It should autogenerate this as `IsA<Widget>` already, like it would do for normal functions. @EPashkin what do you think?

File: Gir.toml (hunk 1 above)
Comment (EPashkin) [reply Gir.toml]: This is property

Comment (EPashkin) [reply Gir.toml]: so something like this:
```
    [[object.property]]
    #need be IsA<Widget>
    name = "attach-widget"
    generate = ["get", "notify"]
```

Comment (sdroege) [reply Gir.toml]: It should however actually generate this correctly automatically, like it would already for normal functions :)

**Comment by EPashkin** [comment]
@sdroege You right, I added issue

**Comment by sdroege** [comment]
Should we merge this now? @EPashkin @GuillaumeGomez ?

Or wait until it's fixed in gir so that we cover all the possible cases at once?

**Comment by GuillaumeGomez** [comment]
I prefer to wait for gir fix.

**Comment by Cogitri** [comment]
Fine by me, will close this in favour of the GIR issue then

**Pull request closed without merging**